docs/cudf/source/basics/groupby.rst

```diff
@@ -128,6 +128,30 @@ Aggregations on groups is supported via the ``agg`` method:
     1   4   1  2.0
     2   5   2  4.5
 
+    >>> gdf
```

Comment:
I'm a little confused. Why did you need to introduce a new DataFrame in this example? Couldn't you just compute the Pearson correlation from the df that's already here?

Reply:
I just liked my example better, but i've added an example using existing dataframe ✔️